tests/unittests/graphOptzTest.cpp

```diff
@@ -604,6 +604,46 @@ TEST_F(GraphOptz, ReshapeNoop) {
   EXPECT_TRUE(SN->getType()->dims().equals(shape));
 }
 
+/// Test on the optimation on the case of reshapeNode following splatNode
```

Comment:
optimation -> optimization

May be replace the line by a more descriptive:
```c++
/// Test the Reshape(Splat(args)) -> Splat(args') transformation
```

Reply:
Done.